Sources/PackageLoading/Platform.swift

```diff
@@ -89,17 +89,17 @@ extension Platform {
     }
 }
 
-public extension Platform {
+extension Platform {
```

Comment:
Isn't this opposite to the pattern we're currently using (making the extension itself public)

Reply:
I remember I've applied the opposite on one of the previous PRs and @tomerd said we prefer access modifiers on separate extension members, not extensions themselves.

Reply:
Ah interesting, I think we should change all the existing code then if that's the preference.

Reply:
SwiftFormat can do that automatically, but it's probably hard to separate that from the rest of it formatting changes. Anyway, I didn't intend to apply them here initially, it's just the header update that forced it in the soundness script.

Reply:
yes having the visibility modifier on the func is safer